modules/integration/tests-common/integration-tests-utils/src/main/java/org/wso2/mb/integration/common/utils/ui/pages/main/DLCContentPage.java

```diff
@@ -63,7 +63,9 @@ public void deleteAllDLCMessages() {
         // Get all the TR elements from the table
         List<WebElement> allDlcRows = dlcTable.findElements(By.tagName("tr"));
 
-            if(allDlcRows.size() > 0) {
+            //The table header row always exists
+            //Therefore, if messages are there in the dlc, the number of rows will be > 1
+            if(allDlcRows.size() > 1) {
```

Comment:
Fixed the indentation in https://github.com/wso2/product-mb/pull/211.
Do we need to flip the if condition since it is a '>' operation?